pkg/controllers/scheduler/framework/plugins/rsp/rsp_test.go

```diff
@@ -40,6 +40,16 @@ func NewFederatedCluster(name string) *fedcorev1a1.FederatedCluster {
 	}
 }
 
+func addTaint(cluster *fedcorev1a1.FederatedCluster, key, value string, effect corev1.TaintEffect) *fedcorev1a1.FederatedCluster {
+	newCluster := cluster
```

Comment:
This is not a copy, why call it `newCluster`?

Reply:
The code has been updated.